src/flamenco/runtime/tests/fd_exec_instr_test.c

```diff
@@ -1406,7 +1423,7 @@ fd_exec_txn_test_run( fd_exec_instr_test_runner_t * runner, // Runner only conta
     /* Capture basic results fields */
     txn_result->executed                          = task_info->txn->flags & FD_TXN_P_FLAGS_EXECUTE_SUCCESS;
     txn_result->sanitization_error                = !( task_info->txn->flags & FD_TXN_P_FLAGS_SANITIZE_SUCCESS );
-    txn_result->has_resulting_state               = false;
+    txn_result->has_resulting_state               = true;
```

Comment:
set this to false for sanitization errors since we dont capture accounts

Reply:
I think this might be a bug of nanopb? I wrote a comment below.
In the tests, if I set it to false I get non-matching results.
Note that this is not a proto field... this is a way that nanopb uses to tell if the field was filled vs not. I thought this was Some(.) vs None in rust, but apparently it isn't...

Reply:
Weird...